railties/test/application/per_request_digest_cache_test.rb

```diff
@@ -12,7 +12,7 @@ class PerRequestDigestCacheTest < ActiveSupport::TestCase
 
   setup do
     build_app
-    add_to_config "config.consider_all_requests_local = true"
+    add_to_env_config "development", "config.action_view.render_hints = false"
```

Comment:
I guess the line `add_to_config "config.consider_all_requests_local = true"` should not be removed?

Reply:
I removed it and the tests still passed.